jenkinsfile_dev

```diff
@@ -2,16 +2,25 @@
 * This pipeline script was created manually based on scripts created by jenkins-build-creator.
 *
 */
-@Library('integration-pipeline-library@master')
+@Library('integration-pipeline-library@ro-declarative-support')
```

Comment:
Will we version this library before merging?

Reply:
So far the Pipeline library hasn't been versioned-- I will be sure to merge my changes back into the master branch though =)